views/partials/protect_yourself.hbs

```diff
@@ -0,0 +1,13 @@
+<h2>Protect yourself from hackers.<br />Start here.</h2>
+<p class="demi">The first step to keeping your online accounts safe is knowing what you’re up against. Enter your email to find out if your accounts have been compromised.</p>
+    <form action="/scan" method="post">
+    <div class="input-group">
+        <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address">
+        <div class="input-group-button">
+            <img src="img/Search.svg" alt="search icon" />
+            <input id="submit-email" type="submit" class="button" value="Search">
+        </div>
+    </div>
+    <span>Your email will not be stored. Find out more in our <a href="
+    https://www.mozilla.org/en-US/privacy/firefox/" alt="Mozilla Firefox privacy policy">Privacy Policy.</a></span>
```

Comment:
nit: we can probably remove the `/en-US/` path in the URL and let the mozilla.org site redirect to the best available locale.